Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -0,0 +1,606 @@
+        /// </summary>
+        /// <param name="cell">The cell that will contain the generated element.</param>
+        /// <param name="dataItem">The data item represented by the row that contains the intended cell.</param>
+        /// <returns>A new <see cref="T:Windows.UI.Xaml.Controls.ComboBox"/> control that is bound to the column's ItemsSource collection.</returns>
+        protected override FrameworkElement GenerateEditingElement(DataGridCell cell, object dataItem)
+        {
+            EnsureColumnBinding(dataItem);
+
+            EnsureDisplayMemberPathExists();
+
+            var comboBox = new ComboBox
+            {
+                Margin = default(Thickness),
+                HorizontalAlignment = HorizontalAlignment.Stretch,
+                VerticalAlignment = VerticalAlignment.Center
+            };
+
+            if (dataItem != null)
+            {
+                var value = dataItem.GetType().GetProperty(Binding.Path.Path).GetValue(dataItem);
+
+                var items = ItemsSource as IEnumerable<object>;
+
+                var selection = !string.IsNullOrEmpty(DisplayMemberPath)
+                    ? items?.FirstOrDefault(x => x.GetType().GetProperty(Binding.Path.Path).GetValue(x).Equals(value))
+                    : items?.FirstOrDefault(x => x.Equals(value));
+
+                comboBox.SelectedItem = selection;
+            }
+
+            var itemsSourceBinding = new Binding
+            {
+                Source = this,
+                Path = new PropertyPath(DATAGRIDCOMBOBOXCOLUMN_itemsSourceName)
+            };
+
+            var displayMemberPathBinding = new Binding
+            {
+                Source = this,
+                Path = new PropertyPath(DATAGRIDCOMBOBOXCOLUMN_displayMemberPathName)
+            };
+
+            comboBox.SetBinding(ComboBox.ItemsSourceProperty, itemsSourceBinding);
+
+            comboBox.SetBinding(ComboBox.DisplayMemberPathProperty, displayMemberPathBinding);
+
+            if (DependencyProperty.UnsetValue != ReadLocalValue(DataGridComboBoxColumn.FontFamilyProperty))
+            {
+                comboBox.FontFamily = this.FontFamily;
+            }
+
+            if (_fontSize.HasValue)
+            {
+                comboBox.FontSize = _fontSize.Value;
+            }
+
+            if (_fontStyle.HasValue)
+            {
+                comboBox.FontStyle = _fontStyle.Value;
+            }
+
+            if (_fontWeight.HasValue)
+            {
+                comboBox.FontWeight = _fontWeight.Value;
+            }
+
+            RefreshForeground(comboBox, (cell != null & cell.OwningRow != null) ? cell.OwningRow.ComputedForeground : null);
+
+            comboBox.SelectionChanged += (sender, args) =>
+            {
+                var item = args.AddedItems.FirstOrDefault();
+                if (item != null)
+                {
+                    var newValue = DisplayMemberPath != null
```

Comment:
DisplayMemberPath != null should be !string.IsNullOrEmpty(this.DisplayMemberPath)

Reply:
Thanks for spotting that